opt/jvm-redis.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
```

Comment:
I think we want to omit this, because it will carry over to other profile.d scripts. /cc @dzuelke

Reply:
I understand the concern. But effectively disallowing to use any bash flags is a bit limiting, especially when they give us more robust scripts. Maybe we can restore previous option flags by storing `$-` before we set ours? I don't see `-o pipefail` in there though.

Reply:
Yes it will carry over to following `.profile.d` scripts, as well as the `Procfile` command, so if a user currently has `cat erpderp | grep matchesonlymaybe; java server.jar`, that'd suddenly crash on boot.

If you need it in a script's structure, you can always wrap the block in question in `(` and `)` to create a subshell; that's the simplest solution (unless you need to export variables :p).